pixi/mocks/mobileFriendlinessCheck/apiResponse.js

```diff
@@ -0,0 +1,59 @@
+const mobileFriendly = {
+  testStatus: {
+    status: 'COMPLETE',
+  },
+  mobileFriendliness: 'MOBILE_FRIENDLY',
+};
+
+const mobileFriendlyWithIssues = {
+  testStatus: {
+    status: 'COMPLETE',
+  },
+  mobileFriendliness: 'MOBILE_FRIENDLY',
+  resourceIssues: [
+    {
+      blockedResource: {
+        url: 'http://www.example.com/resource1.jpg',
+      },
+    },
+    {
+      blockedResource: {
+        url: 'http://www.example.com/resource2.jpg',
+      },
+    },
+  ],
+};
+
+const notMobileFriendly = {
+  testStatus: {
+    status: 'COMPLETE',
+  },
+  mobileFriendliness: 'NOT_MOBILE_FRIENDLY',
+};
+
+const unspecifiedStatus = {
+  testStatus: {
+    status: 'TEST_STATUS_UNSPECIFIED',
+  },
+};
+
+const internalError = {
+  testStatus: {
+    status: 'INTERNAL_ERROR',
+  },
```

Comment:
These should still have `mobileFriendliness: 'MOBILE_FRIENDLY_TEST_RESULT_UNSPECIFIED'` iiuc? Talked with @nainar and we settled on this resolving to `Manual check` similar to Intrusive Interstitials.